tests/unit/Rules/PositiveTest.php

```diff
@@ -13,65 +13,54 @@
 
 namespace Respect\Validation\Rules;
 
-use PHPUnit\Framework\TestCase;
+use Respect\Validation\Test\RuleTestCase;
 
 /**
  * @group  rule
+ *
  * @covers \Respect\Validation\Rules\Positive
  * @covers \Respect\Validation\Exceptions\PositiveException
```

Comment:
Please remove this `@covers \Respect\Validation\Exceptions\PositiveException` exceptions are tested on integrations test